examples/05_transformers.ipynb

```diff
@@ -0,0 +1,851 @@
+   "metadata": {},
+   "source": [
+    "Generally, the series passed to `transform` need not be the same as in `fit` - if it is, the shorthand `fit_transform` can instead be used:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_boxcox_trafo.fit_transform(X_series)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The transformer can also be applied to `Panel` data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_boxcox_trafo.fit_transform(X_panel)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Example: transforming series to primitive features\n",
+    "\n",
+    "The summary transformer can be used to extract sample statistics such as mean and variance from a series. First, we construct the transformer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# constructing the transformer\n",
+    "from sktime.transformations.series.summarize import SummaryTransformer\n",
+    "\n",
+    "my_summary_trafo = SummaryTransformer()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As before, we can fit/apply with `fit`, `transform`, and `fit_transform`.\n",
+    "\n",
+    "`SummaryTransformer` returns primitive features, hence the output will be a `pandas.DataFrame`, each row corresponding to one series in the input. \n",
+    "\n",
+    "If the input is a single series, the output of `transform` and `fit_transform` will be a one-row `DataFrame`, corresponding to mean etc of that one series:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_summary_trafo.fit_transform(X_series)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the input is a panel, the output of `transform` and `fit_transform` will be a `DataFrame` with as many rows as the `Panel` had series. The row with the corresponding index has mean etc of the series in the panel `X_panel` with the same index:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_summary_trafo.fit_transform(X_panel)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### transformers with series/panel output vs primitive output\n",
+    "\n",
+    "Whether a transformer will return primitives, i.e., a `pandas.DataFrame` in `transform`, or time series like objects (`Series` or `Panel`), can be checked by using the `\"scitype:transform-output\"` tag. This is `\"Series\"` for behaviour as in the first example (box-cox), and `\"Primitives\"` for behaviour as in the second example (summarizer):"
```

Comment:
* minor restructuring of first sentence to hopefully make clearer, e.g. putting 'ie a dataframe' into brackets, reordering so that time series comes before primitives (to match the second sentence)
* explicitly naming the example transformers used
```suggestion
    "Whether `transform` will return time series like objects (`Series` or `Panel`) or primitives (i.e. a `pandas.DataFrame`) can be checked by using the `\"scitype:transform-output\"` tag. This is `\"Series\"` for behaviour as in the first example (`BoxCoxTransformer`), and `\"Primitives\"` for behaviour as in the second example (`SummaryTransformer`):"
```